static/src/js/pages/KeywordManagerPage/__tests__/KeywordManagerPage.test.jsx

```diff
@@ -328,6 +332,232 @@ describe('KeywordManagerPage component', () => {
     })
   })
 
+  describe('when publishing a new version', () => {
+    let publishKmsConceptVersionMock
+
+    beforeEach(() => {
+      publishKmsConceptVersionMock = vi.fn()
+      vi.spyOn(publishKmsConceptVersionModule, 'publishKmsConceptVersion').mockImplementation(publishKmsConceptVersionMock)
+    })
+
+    test('should close publish modal when toggleModal is called', async () => {
+      const { user } = setup()
+
+      // Open the publish modal
+      const publishButton = screen.getByRole('button', { name: /publish new keyword version/i })
+      await user.click(publishButton)
+
+      // Check if the modal is open by looking for the version name input
+      await waitFor(() => {
+        expect(screen.getByLabelText('Version Name:')).toBeInTheDocument()
```

Comment:
```suggestion
        expect(screen.getByLabelText('Version Name:')).toBeVisible()
```
Check that the hidden is false instead of just on the DOM

Reply:
Done